### PR TITLE
RGO production and price changes

### PR DESCRIPTION
**Pull request by unique-usernames-are-annoying**

### CODE REVIEW

File: common/production_types.txt [hunk 1]
```diff
@@ -456,14 +456,14 @@ RGO_template_clerks = {
...
-	value = 3
+	value = 6.5
 	mine = yes
 }
 
 food_fields = {
 	template = RGO_template_farmers
 	output_goods = food
-	value = 7.0
+	value = 7.5
```
Comment (SirRunner) [inline common/production_types.txt:466]: I thought the goal was to cut down on the amount of food being produced?

File: common/production_types.txt [hunk 2]
```diff
@@ -477,35 +477,35 @@ stables = {
 artisinal_workshop = {
 	template = RGO_template_clerks
 	output_goods = artisinal_goods
-	value = 1
+	value = 3.9
```
Comment (SirRunner) [inline common/production_types.txt:480]: Is this intentional? Artisinal goods is not supposed to be as profitable as precious metals. Right off the start I can see that the labourers are filling all of their life, most of their everyday and some of their luxury needs, and the nobles are right there with them too

File: common/production_types.txt [hunk 3]
```diff
@@ -477,35 +477,35 @@ stables = {
...
-	value = 2.4
+	value = 7
 	mine = yes
 }
 
 cotton_plantation = {
 	template = RGO_template_farmers
 	output_goods = cotton
-	value = 2.2
+	value = 7.9
```
Comment (SirRunner) [inline common/production_types.txt:494]: I think this is a little high. Originally I was only putting Cotton in Harad, but now that it's been expanded to Khand, Mordor, Gondor and Rohan, the shortage of cotton can partially be resolved by putting more cotton RGOs around

File: common/production_types.txt [hunk 4]
```diff
@@ -841,22 +841,22 @@ artisan_dye = {
 
 ##### ARTISAN ONLY #####
 
-artisan_horse_to_food = {
```
Comment (SirRunner) [inline common/production_types.txt:844]: Note that these artisan cycles are there to help keep the economy moving -- I put these in at the recommendation of the developers of IgorPuir, who have a much better understanding of the economy than I do. We will almost certainly want to put them back

**Review by SirRunner — COMMENTED** [review]
A lot of these increases in RGO production seem massive (3x - 4x). Is that intentional?

I can understand the ones that are locked to hill/mountain provinces (iron, coal, wool, rock), but do question the rest

**Comment by unique-usernames-are-annoying** [comment]
the main intention of this is to make the possible income for labourer jobs almost equal, the next step is to then adjust production and prices inversely depending on where the supply is too high or low as well as demand from pops. for now the changes would make all jobs earn 15 to 16 at 0% unemployment. lets see how bad formatting for this ends up:

RGO			production	* price = total value
precious metal	2			* 8		= 16
cotton			7,9			* 2		= 15,8
horses			2			* 7,9	= 15,8
artisinal_goods	3,9			* 4		= 15,6
rock				6,5			* 2,4	= 15,6
tea				6			* 2,6	= 15,6
dye				1,3			* 12		= 15,6
iron				4,4			* 3,5	= 15,4
coal				7			* 2,2	= 15,4
tobacco			14			* 1,1	= 15,4
wool			22			* 0,7	= 15,4
timber			8,5			* 1,8	= 15,3
food			7,5			* 2		= 15

**Review by SirRunner — APPROVED** [review]

**Pull request merged**